sos/plugins/dnf.py

```diff
@@ -60,4 +66,17 @@ def setup(self):
         if self.get_option("history"):
             self.add_cmd_output("dnf history")
 
+        if self.get_option("history-info"):
+            history = self.call_ext_prog("dnf history")
```

Comment:
Ah, I was partially blind.. Doesn't it makes sense to implicitly enable `history` option when `history-info` is enabled?